testsuite/tests/kuadrant/authorino/metrics/test_deep_metrics.py

```diff
@@ -56,5 +56,5 @@ def test_deep_metrics(metric_name, metric_type, deep_metrics):
         lambda x: x["metric"]["evaluator_name"] == metric_name and x["metric"]["evaluator_type"] == metric_type
     )
 
-    assert metrics
+    assert len(metrics.metrics) > 0
```

Comment:
in which scenario can it be more than one?

Reply:
If one of the AuthConfig sections was hit two times